build_defs/go.build_defs

```diff
@@ -512,6 +515,12 @@ def go_library(name:str, srcs:list, resources:list=[], asm_srcs:list=None, hdrs:
         exported_deps = exported_deps,
     )
 
+
+def _goexperiment_env():
+    """Returns the GOEXPERIMENT env var, or None if there isn't one."""
+    return {"GOEXPERIMENT": CONFIG.GO.EXPERIMENT} if CONFIG.GO.EXPERIMENT else None
```

Comment:
If this were to return `{}` rather than `None`, the return value could unconditionally be concatenated with all instances of `env` parameters modified in this PR.